config/job.yaml.tim

```diff
@@ -39,9 +44,25 @@ spec:
         volumeMounts:
         - mountPath: /opt/screwdriver
           name: screwdriver
+        - mountPath: /var/run/sd
+          name: logs
```

Comment:
Since it is just /var/run/sd, a more appropriate name might be `runpath` or something. A better name for `socket` in that directory might be `logsocket`